### ISSUE TITLE
Fatal error while cloning the git repo

### ISSUE THREAD

**Issue by YashAgarwal** [dabit3/react-native-hackathon-starter#50]
```bash
git clone --recursive https://github.com/react-native-training/react-native-elements-app.git
```
```bash
Cloning into 'react-native-elements-app'...
remote: Counting objects: 1308, done.
remote: Compressing objects: 100% (51/51), done.
remote: Total 1308 (delta 57), reused 89 (delta 43), pack-reused 1198
Receiving objects: 100% (1308/1308), 6.00 MiB | 336.00 KiB/s, done.
Resolving deltas: 100% (653/653), done.
Submodule 'node_modules/react-native-elements' (git@github.com:react-native-training/react-native-elements.git) registered for path 'node_modules/react-native-elements'
Cloning into '/Users/yashagarwal/arti/react-native-elements-app/node_modules/react-native-elements'...
Warning: Permanently added the RSA host key for IP address '13.229.188.59' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:react-native-training/react-native-elements.git' into submodule path '/Users/yashagarwal/arti/react-native-elements-app/node_modules/react-native-elements' failed
Failed to clone 'node_modules/react-native-elements'. Retry scheduled
Cloning into '/Users/yashagarwal/arti/react-native-elements-app/node_modules/react-native-elements'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:react-native-training/react-native-elements.git' into submodule path '/Users/yashagarwal/arti/react-native-elements-app/node_modules/react-native-elements' failed
Failed to clone 'node_modules/react-native-elements' a second time, aborting
```

**Comment by xavier-villelegier** [comment]
Thanks for reporting this @YashAgarwal, we'll look into it. In the meantime you can manually clone `react-native-elements` repo inside `node_modules`, or just do an `npm i`, you'll have the error of the submodule but you should have the original RNE from npm

**Comment by gianpaj** [comment]
I don't know about the permissions issue. I think that's more an SSH key issue with Github

But there's issue fetching a specific commit:

```
git clone --recursive https://github.com/react-native-training/react-native-elements-app.git
Cloning into 'react-native-elements-app'...
remote: Counting objects: 1272, done.
remote: Compressing objects: 100% (80/80), done.
remote: Total 1272 (delta 56), reused 71 (delta 27), pack-reused 1157
Receiving objects: 100% (1272/1272), 6.12 MiB | 1.28 MiB/s, done.
Resolving deltas: 100% (621/621), done.
Checking connectivity... done.
Submodule 'node_modules/react-native-elements' (git@github.com:react-native-training/react-native-elements.git) registered for path 'node_modules/react-native-elements'
Cloning into react-native-elements-app/node_modules/react-native-elements'...
Fetched in submodule path 'node_modules/react-native-elements', but it did not contain b5e7dee93fa86d219a5e1754c40fca9ae1d2002f. Direct fetching of that commit failed.
```

link to commit:
https://github.com/react-native-training/react-native-elements/commit/b5e7dee93fa86d219a5e1754c40fca9ae1d2002f

**Comment by xavier-villelegier** [comment]
@gianpaj You can still clone manually the module inside the `node_modules` dir, same result in the Example app. We have to think of a way to make this easier to get live reload, for now this is the easiest we came up with

**Comment by esbenvb** [comment]
Fixed in #59
